packages/client/hmi-client/src/components/operator/tera-operator-outputs.vue

```diff
@@ -107,4 +160,14 @@ li > *:not(:first-child) {
 	top: -0.35rem;
 	left: -0.35rem;
 }
+
+.v-enter-active,
+.v-leave-active {
+	transition: opacity 0.5s ease;
+}
+
+.v-enter-from,
+.v-leave-to {
+	opacity: 0;
+}
```

Comment:
You may want to add a comment here because these v-*** classes are seemingly not being used and someone might accidentally clean them out.

Reply:
added a comment here now!